### PR TITLE
fix: use new across config store instead of rate model store

### PR DESCRIPTION
**Pull request by daywiss**
Signed-off-by: David <david@umaproject.org>

This basically replaces references to rate model store with the config store.

### CODE REVIEW

**Review by amateima — APPROVED** [review]
👍

File: src/contracts/acrossConfigStore.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { object, string, Infer, assert } from "superstruct";
+import type { CallOverrides } from "@ethersproject/contracts";
+
+const RateModel = object({
+  UBar: string(), // denote the utilization kink along the rate model where the slope of the interest rate model changes.
+  R0: string(), // is the interest rate charged at 0 utilization
+  R1: string(), // R_0+R_1 is the interest rate charged at UBar
+  R2: string(), // R_0+R_1+R_2 is the interest rate charged at 100% utilization
+});
+const L1TokenConfig = object({
```
Comment (mrice32) [inline src/contracts/acrossConfigStore.ts:12]: nit: I don't love aliasing variables and types. When reading I kept thinking you were passing the RateModel and L1TokenConfig _types_ into functions. feel free to ignore if this is the way the library suggests you name things, just felt a little odd to me

**Review by mrice32 — APPROVED** [review]
LGTM!

File: src/contracts/acrossConfigStore.ts (hunk 1 above)
Comment (daywiss) [reply src/contracts/acrossConfigStore.ts:12]: ok sure i can adjust that, not sure whats recommended here

**Pull request merged**